cypress/support/pages/settingPageObjects.js

```diff
@@ -0,0 +1,69 @@
+import SignInPageObject from "./signIn.pageObject";
+
+class SettinsPageObject extends SignInPageObject {
+  url = '/settings';
```

Comment:
There seems to be a typo in the class name. It should be SettingsPageObject instead of SettinsPageObject.